apollo-router/src/services/mod.rs

```diff
@@ -122,8 +122,16 @@ impl RouterRequest {
         variables: HashMap<String, Value>,
         extensions: HashMap<String, Value>,
         context: Option<Context>,
-        headers: MultiMap<IntoHeaderName, IntoHeaderValue>,
+        mut headers: MultiMap<IntoHeaderName, IntoHeaderValue>,
```

Comment:
This is a breaking change and should be noted in the changelog. I honestly don't think anyone will be affected by this change, but we should document it.

Something like:
With this change, if a request does not specify a `content-type`, then a default `content-type` of `application/json` will be provided.

Reply:
Done